src/pages/Product/components/Review/Review.js

```diff
@@ -13,13 +15,20 @@ const Review = () => {
       </div>
       <div className="reviewTitle">배송도 빠르고 정말 좋아요.</div>
       <div className="reviewMain">
-        <div className="reviewMainContents">
-          주문한지 하루만에 배송이 도착해서 너무 좋았습니다.
-          <br />
-          다음에도 필요하면 또 여기서 시킬 것 같아요.
-        </div>
-        <span className="reviewMainHandler" onClick={() => {}}>
-          더보기 ▼
+        {isMore === true ? (
```

Comment:
```suggestion
        {isMore ? (
```
isMore 값 자체가 boolean 값이기에 조건문 형태로 구현하지 않으셔도 됩니다!